.github/workflows/tests_e2e_ios.yml

```diff
@@ -129,6 +129,13 @@ jobs:
       - name: Start Firestore Emulator
         run: yarn tests:emulator:start-ci
 
+      # https://bitrise.io/blog/post/xcode-15-performance-regressions
```

Comment:
ludicrous this is necessary

Reply:
Yeah the simulator was taking quite some time to be ready, sometimes 10mins, this seemed to help significantly but it is still fairly slow sometimes 🥲